src/inheritAttributes.js

```diff
@@ -215,6 +216,68 @@ export const parseCaptionServiceMetadata = (service) => {
 
       return {channel, language};
     });
+  } else if (service.schemeIdUri === 'urn:scte:dash:cc:cea-708:2015') {
+    const values = service.value.split(';');
+
+    return values.map((value) => {
+      // service or channel number 1-63
+      let channel;
+      // language is a 3ALPHA per ISO 639.2/B
+      // field is required
+      let language;
+      // BIT 1/0 or ?
+      // default value is 1, meaning 16:9 aspect ratio, 0 is 4:3, ? is unknown
+      let aspectRatio = 1;
+      // BIT 1/0
+      // easy reader flag indicated the text is tailed to the needs of beginning readers
+      // default 0, or off
+      let easyReader = 0;
+      // BIT 1/0
+      // If 3d metadata is present (CEA-708.1) then 1
+      // default 0
+      let threed = 0;
```

Comment:
If we do go with a `flags` object we could make this `3d` as `threed` just looks like a mis-spelling to me.